packages/renderer/src/lib/toast/ToastCustomUi.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+import { faCheckCircle, faCircleExclamation, faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { toast } from '@zerodevx/svelte-toast';
+import Fa from 'svelte-fa';
+
+import type { TaskInfo } from '/@api/taskInfo';
+
+let { toastId, taskInfo, onpop = () => {} }: { toastId: number; taskInfo: TaskInfo; onpop?: () => void } = $props();
```

Comment:
```suggestion
interface Props {
    toastId: number;
    taskInfo: TaskInfo;
    onpop?: () => void;
}

let { toastId, taskInfo, onpop = () => {} }: Props = $props();
```

I feel like it is easier to read and understand the components props